HISTORY.rst

```diff
@@ -25,6 +25,60 @@ unreleased
   are supported. Once registered, the device will show up in the linked IoT Hub and can be interacted with or
   simulated using other `az iot device` commands.
 
+**IoT Central updates**
+
+* Add support for device groups CRUD.
+  - az iot central device-group
+    - az iot central device-group list
+    - az iot central device-group show
+    - az iot central device-group create
+    - az iot central device-group delete
+    - az iot central device-group update
+
+* Add support for device attestation CRUD.
+  - az iot central device attestation
+    - az iot central device attestation show
+    - az iot central device attestation create
+    - az iot central device attestation delete
+    - az iot central device attestation update
+
+* Add support for device/module properties/telemetry/command.
+  - az iot central device list-components
+  - az iot central device telemetry
+    - az iot central device telemetry show
+  - az iot central device component-telemetry
+    - az iot central device component-telemetry show
+  - az iot central device properties
```

Comment:
As we discussed, we shouldn't expose different `properties` concept from central, let's algin with `twin`, using following command to address get `twin` from device, device module, device component, device module component: ``` az iot central device twin show --app-id [appid] --device-id [deviceid] --module-name [modulename] --component-name [componentname]```

Reply:
Right, working on this.